src/cms/forms/pages/page_translation_form.py

```diff
@@ -137,8 +138,24 @@ def clean_text(self):
 
         if "<h1>" in text:
             raise ValidationError(
-                _("Use of Heading 1 style not allowed."),
+                _("Use of Heading 1 style is not allowed."),
                 code="no-heading-1",
             )
 
+    def clean_text2(self):
+        text = self.data["text"]
+
+        # r'href=[\'"]?([^\'" >]+)'
+        # r(?<=href=").*?(?=")+.*?(?<=target=")
+        r = re.compile(r"(\B|\b)<a+.+\/a>(\B|\b)")
```

Comment:
```suggestion
        r = re.compile(r"target=[\"']?_blank[\"']?")
```
I'd suggest the following regex. We only want to prevent `target="_blank"` (with all different possibilities regarding quotation marks). It should be allowed to use normal links with `<a>`-tags.